handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -725,8 +725,11 @@ private void finishWrap(ChannelHandlerContext ctx, ByteBuf out, ChannelPromise p
         }
     }
 
-    // This method will not call setHandshakeFailure(...) !
-    private void wrapNonAppData(ChannelHandlerContext ctx, boolean inUnwrap) throws SSLException {
+    /**
+     * This method will not call setHandshakeFailure(...) !
```

Comment:
@Scottmitch as you changed this to java docs please use `{@link #setHandshakeFailure(.....`